examples/sentence-transformers-training/paraphrases/readme.md

```diff
@@ -0,0 +1,40 @@
+
+# Paraphrases Training
+
+## Requirements
+
+First, you should install -
+```bash
+pip install -U sentence-transformers
+pip install git+https://github.com/huggingface/optimum-habana.git
+```
+
+
+## Usage
+
+To training on Paraphrases -
+
+```bash
+python examples/sentence-transformers-training/paraphrases/training_paraphrases.py 
```

Comment:
Can we update this to provide other models from HF as commandline argument.

Reply:
done